### PR TITLE
59 Fixing paginate-v2 and github pages, second approach

### PR DESCRIPTION
**Pull request by avcconti**
Since the first approach fo leaving "_site" folder alive from .gitignore did not work as expected, the second approach focuses on using the option "docs folder" that github pages allows in config. This new folder has been tested in my personal GHPages and worked as expected.

### CODE REVIEW

File: docs/feed.xml [hunk 1]
```diff
@@ -16,9 +16,9 @@ eran risas y pegatinas?&lt;/p&gt;
 
 </description>
         
-        <pubDate>Sun, 10 Nov 2013 04:18:00 -0600</pubDate>
-        <link>https://gdgtoledo.github.io/web/posts/2013-11-10-hola-mundo/</link>
-        <guid isPermaLink="true">https://gdgtoledo.github.io/web/posts/2013-11-10-hola-mundo/</guid>
+        <pubDate>Sun, 10 Nov 2013 10:18:00 +0000</pubDate>
+        <link>http://0.0.0.0:4000/posts/2013-11-10-hola-mundo/</link>
```
Comment (mdelapenya) [inline docs/feed.xml]: ¿Es correcta esta IP?

File: docs/feed.xml [hunk 2]
```diff
@@ -3,8 +3,8 @@
   <channel>
     <title>GDG Toledo</title>
     <description>El espíritu de este grupo es fomentar y compartir las buenas prácticas y las experiencias en herramientas, frameworks, lenguajes, etc</description>
-    <link>https://gdgtoledo.github.io/web</link>
-    <atom:link href="https://gdgtoledo.github.io/web/feed.xml" rel="self" type="application/rss+xml" />
+    <link>http://0.0.0.0:4000</link>
```
Comment (mdelapenya) [inline docs/feed.xml]: Lo mismo aquí

File: docs/feed.xml (hunk 1 above)
Comment (avcconti) [reply docs/feed.xml]: No, la generación del feed.xml debe estar cogiendo valores de entorno local.

Comment (avcconti) [reply docs/feed.xml]: Fixed

File: docs/feed.xml (hunk 2 above)
Comment (avcconti) [reply docs/feed.xml]: Fixed

**Comment by javierlopezdeancos** [comment]
approve a ciegas no tengo ni papa de aque hace esta PR

**Pull request merged**